Adyen/Core/AdyenContext/AdyenContext.swift

```diff
@@ -18,8 +20,9 @@ public final class AdyenContext: PaymentAware {
     /// The payment information.
     public private(set) var payment: Payment?
     
-    @_spi(AdyenInternal)
-    public let analyticsProvider: AnyAnalyticsProvider?
+    package let analyticsProvider: AnyAnalyticsProvider?
+    
+    package let amount: Amount
```

Comment:
This can be public, right?

Reply:
Let's start with restricted access and can switch to public if needed